src/MayaraServer.cpp

```
@@ -420,7 +432,33 @@ bool MayaraServer::Start() {
+  // we are unloaded (see above). On POSIX `exec` replaces the shell, so m_pid
+  // stays the server's own; cmd.exe has no such trick and waits for it, which
+  // Running() (it exits when the server does) and Stop() (wxKILL_CHILDREN)
+  // both cope with. The quoting is the shell's, inside the one argument
+  // wxExecute hands it: single quotes on POSIX, since a double quote there
+  // would end that argument early; cmd.exe instead takes the whole line in an
+  // outer pair of its own and strips it.
+  const wxString log = LogPath();
+#ifdef _WIN32
+  const wxString launch = "cmd.exe /c \"" + cmd + " > \"" + log + "\" 2>&1\"";
+#else
+  const wxString launch = "/bin/sh -c \"exec '" + BinaryPath() + "'" + args +
```

Clang (14.0.6)

[warning] variable 'launch' is not initialized

(cppcoreguidelines-init-variables)